code/src/components/TweetForm.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+const TweetForm = ({ newTweet, onNewTweetChange, handleFormSubmit }) => {
+  return (
+    <div className="form-container">
+      <form onSubmit={handleFormSubmit}>
+        <h1 className="title">Spread some love!</h1>
```

Comment:
Just nitty gritty design feedback... The heading is centered but not all the way in the middle of the desktop version. I would add `text-align: center;` to solve that, or remove the padding and make it left-aligned like the mobile version.

Reply:
Ah, sloppy me!! Thank you.